README.md

```diff
@@ -1,2 +1,91 @@
-# inferno-backend
+# Inferno backend
+
 The backend of the Inferno Rewards project. It's intended to determine the awarded addresses and to distribute the weekly rewards to them.
+
+## Setup
+
+Install dependencies
+
+ - [nvm](https://github.com/nvm-sh/nvm/blob/master/README.md)
+
+Then run:
+
+```sh
+$ nvm install && nvm use
+$ npm add -g yarn
+$ yarn && yarn build
+```
+
+At this point you will have a directory at `./lib`, there will be the compiled JS files.
+
+## Cache
+
+The folder `./cache` contains the immutable cached blocks. Every time you run the `aggregate` command, the non-cached blocks will be added there.
```

Comment:
We could explain better that the cache is from gql queries

Reply:
b6f28a9991406ba698b6b8e24886219ae56d62d8